CHANGELOG.md

```diff
@@ -1,9 +1,17 @@
 # Changelog
 
-## 8.9.1
+## 8.9.2
```

Comment:
Would this need to be a major version bump because we're changing Map to Set [here](https://github.com/spotify/docker-client/pull/898/files#diff-30a967cd2986f001106d3df2023eb161R113)?

Reply:
I think you’re right. I had thought that I kept the API the same and just shuffled things around internally, but this is an API change.

Should I bump this line and the version in `pom.xml` up to `8.10.0`?

Reply:
I guess, regardless of what I change the version to, I cannot make it `8.9.2`. That version has already been released.